scripts/build-storybooks.js

```diff
@@ -119,9 +119,14 @@ const createContent = deployables => {
 const handleExamples = async files => {
   const deployables = files.filter(f => {
     const packageJsonLocation = p(['examples', f, 'package.json']);
-    const stats = statSync(packageJsonLocation);
-
-    return stats.isFile() && hasBuildScript(packageJsonLocation);
+    let stats = null;
+    try {
+      stats = statSync(packageJsonLocation);
+    } catch (e) {
+      //
```

Comment:
log the error, or at least a summary

Reply:
The error it's hiding is when there are temp system files in the examples directory, which are not real examples.

I do not think it's productive to see those errors tbh.

WDYT?

Reply:
can you just print out: `Skipping: ${e.message}` or something?